app/components/works/detail_component.html.erb

```diff
@@ -238,5 +238,5 @@
       <% end %>
     </tbody>
   </table>
-  <%= render Works::WorkTypeModalComponent.new %>
+  <%= render Works::WorkTypeModalComponent.new(turbo: false, method: :patch, form_authenticity_token:) %>
```

Comment:
Setting turbo to false since in a frame. This then requires setting the method and providing a form authenticity token so that the form works correctly.